internal/yunikorn/rest_test.go

```diff
@@ -1,22 +1,40 @@
-package ykclient
+package yunikorn
```

Comment:
This feels a bit strange - we're not directly part of the Yunikorn project. But `yhs` and `yunikorn-history-server` are not great either.

Reply:
So you have a package called `yunikon` in which you have an generic `Client` which defines Yunikorn API methods, a `RESTClient` which calls Yunikorn API via REST and a `Service` which does some async work like syncing data with Yunikorn and tapping into the Event Stream.